File: src/com/eagle/programmar/Basic/Statements/Basic_RemStatement.java
// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, May 8, 2025

package com.eagle.programmar.Basic.Statements;

import com.eagle.interpret.EagleInterpreter;
import com.eagle.interpret.EagleRunnable;
import com.eagle.programmar.Basic.Terminals.Basic_Comment;
import com.eagle.programmar.Basic.Terminals.Basic_Keyword;
import com.eagle.tokens.TokenSequence.S;
import com.eagle.tokens.interfaces.AbstractStatement;

public class Basic_RemStatement extends Basic_Comment
		implements EagleRunnable, AbstractStatement
{
	public @S(10) Basic_Keyword REM = new Basic_Keyword("REM");
	public @S(20) Basic_Comment comment;

	@Override
	public void interpret(EagleInterpreter interpreter)
	{
		// Nothin' to do
	}
}

File: csrc/com/eagle/programmar/Basic/Statements/Basic_RemStatement.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, May 8, 2025

namespace com.eagle.programmar.Basic.Statements
{
	using EagleInterpreter = com.eagle.interpret.EagleInterpreter;
	using EagleRunnable = com.eagle.interpret.EagleRunnable;
	using Basic_Comment = com.eagle.programmar.Basic.Terminals.Basic_Comment;
	using Basic_Keyword = com.eagle.programmar.Basic.Terminals.Basic_Keyword;
	using TokenSequence = com.eagle.tokens.TokenSequence;
	using AbstractStatement = com.eagle.tokens.interfaces.AbstractStatement;

	public class Basic_RemStatement : TokenSequence, EagleRunnable, AbstractStatement
	{
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(10) com.eagle.programmar.Basic.Terminals.Basic_Keyword REM = new com.eagle.programmar.Basic.Terminals.Basic_Keyword("REM");
		public Basic_Keyword REM = new Basic_Keyword("REM");
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(20) com.eagle.programmar.Basic.Terminals.Basic_Comment comment;
		public Basic_Comment comment;

		public override void interpret(EagleInterpreter interpreter)
		{
			// Nothin' to do
		}
	}

}
